modules/API/API.js

```diff
@@ -43,198 +36,239 @@ let postis;
+     * @returns {void}
      */
-    notifyUserJoined (id) {
-        triggerEvent("participant-joined", {id});
+    notifyUserJoined(id) {
+        triggerEvent('participant-joined', { id });
     }
 
     /**
      * Notify external application (if API is enabled) that
      * user left the conference.
-     * @param {string} id user id
+     *
+     * @param {string} id - User id.
+     * @returns {void}
      */
-    notifyUserLeft (id) {
-        triggerEvent("participant-left", {id});
+    notifyUserLeft(id) {
+        triggerEvent('participant-left', { id });
     }
 
     /**
      * Notify external application (if API is enabled) that
      * user changed their nickname.
-     * @param {string} id user id
-     * @param {string} displayName user nickname
+     *
+     * @param {string} id - User id.
+     * @param {string} displayName - User nickname.
+     * @returns {void}
      */
-    notifyDisplayNameChanged (id, displayName) {
-        triggerEvent("display-name-change", {id, displayname: displayName});
+    notifyDisplayNameChanged(id, displayName) {
+        triggerEvent('display-name-change', { id,
+            displayname: displayName });
     }
 
     /**
      * Notify external application (if API is enabled) that
-     * user changed their nickname.
-     * @param {string} id user id
-     * @param {string} displayName user nickname
+     * the conference has been joined.
+     *
+     * @param {string} room - The room name.
+     * @returns {void}
      */
-    notifyConferenceJoined (room) {
-        triggerEvent("video-conference-joined", {roomName: room});
+    notifyConferenceJoined(room) {
+        triggerEvent('video-conference-joined', { roomName: room });
     }
 
     /**
      * Notify external application (if API is enabled) that
      * user changed their nickname.
-     * @param {string} id user id
-     * @param {string} displayName user nickname
+     *
+     * @param {string} room - User id.
+     * @param {string} displayName - User nickname.
+     * @returns {void}
      */
-    notifyConferenceLeft (room) {
-        triggerEvent("video-conference-left", {roomName: room});
+    notifyConferenceLeft(room) {
+        triggerEvent('video-conference-left', { roomName: room });
     }
 
     /**
      * Notify external application (if API is enabled) that
      * we are ready to be closed.
+     *
+     * @returns {void}
      */
-    notifyReadyToClose () {
-        triggerEvent("video-ready-to-close", {});
+    notifyReadyToClose() {
+        triggerEvent('video-ready-to-close', {});
     }
 
     /**
      * Sends remote control event.
-     * @param {RemoteControlEvent} event the remote control event.
+     *
+     * @param {RemoteControlEvent} event - The remote control event.
+     * @returns {void}
      */
     sendRemoteControlEvent(event) {
-        sendMessage({method: "remote-control-event", params: event});
+        sendMessage({ method: 'remote-control-event',
```

Comment:
I'm not sure about the line breaking. I looked around jitsi-meet and looks like the common style is to break after the brackets instead of after the first item, and if line breaks are used put the closing brackets on their own line.
```
sendMessage({
    method: 'remote-control-event'
    params: event
});
```